docs/conf.py

```diff
@@ -29,7 +29,11 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = []
+extensions = [
+    'sphinx.ext.intersphinx',
+]
+
+intersphinx_mapping = {'python': ('https://docs.python.org/3.5', None)}
```

Comment:
Should probably use `https://docs.python.org/3` as the link so that the links are always against the latest release instead of pegging against 3.5.